docs/guide/types-of-values.md

```diff
@@ -30,6 +30,28 @@ operations such as calculating the number of days between two dates.
 - A DateTime value is represented as the number of (possibly fractional) days
   since [`nullDate`](../api/interfaces/configparams.md#nulldate).
 
+## Forcing the string value type
+
+Similarly to other popular spreadsheet software, HyperFormula automatically detects the type of an input value.
+On some occasions, the value should be treated as a string even though it's parsable as a number/date/formula/etc.
```

Comment:
```suggestion
On some occasions, the value should be treated as a string even though it's parsable as a number, date, formula, etc.
```

Reply:
Also wondering about the "etc." can we be more explicit?

Reply:
Changed to the complete list of cell value types.